### ISSUE TITLE
Check publication of translations

### ISSUE THREAD

**Issue by clochix** [cozy/cozy-contacts#269]
Found int he logs of a german user:
```
["WARNING: Missing translation for key: \"card actions add share\""]
```

Seems like the german locale is missing in the build files.

**Comment by Phyks** [comment]
Same with a French cozy instance (cozy-contacts v2.1.11).

**Comment by Phyks** [comment]
Also, same with "datapoint placeholder label", see screenshot.

<img width="510" alt="capture d ecran 2016-09-13 a 17 56 59" src="https://cloud.githubusercontent.com/assets/3856586/18481420/6b9ca8c0-79dc-11e6-88a8-5c964bb1fc22.png">

**Comment by gregorylegarec** [comment]
Thanks for your feedback @Phyks.

All those points seem to be fixed in version 2.1.14.

**Comment by Phyks** [comment]
Indeed, thanks!
